array_diff.py

```diff
@@ -20,28 +37,9 @@
 pop2 = [2]
 pop_array = [1, 2]
 
-def array_diff(original_array,query_array):
-    #initiate an empty array
-    new_array = []
-    i = 0
-    if query_array == [] or original_array == []:
-        return original_array
-    else:
-        for y in query_array:
-             #check to see if is in list 2 by iterating through list 2 with equality          
-            for x in original_array:
-                #if it is in there, pop it
-                # cant use if x != y because you'll 
-                if x not in query_array and y not in new_array:
```

Comment:
The in keyword can be thought of as syntactic sugar for another for loop.  There is no reason to have 3 nested for loops here, it can be accomplished with two.